src/main/java/uk/gov/hmcts/reform/coh/controller/DecisionController.java

```diff
@@ -39,11 +38,14 @@
 
     private DecisionStateService decisionStateService;
 
+    private DecisionStateHistoryService decisionStateHistoryService;
```

Comment:
remove decisionStateHistoryService because is never used

Reply:
Done